topo/node/ixia/ixia.go

```diff
@@ -59,6 +59,11 @@ func (n *Node) newCRD() *ixiatg.IxiaTG {
 		},
 	}
 
+	if n.GetProto().Config != nil {
+		ixiaCRD.Spec.InitContainer.Image = n.GetProto().Config.InitImage
+		ixiaCRD.Spec.InitContainer.Sleep = n.GetProto().Config.Sleep
```

Comment:
sleep should be the sleep time before starting the pod according to the documentation: https://github.com/openconfig/kne/blob/1775c426748ac4100544b08c9383709da53593fe/proto/topo.proto#L125

Does this instead create the pod without sleeping but then start the initcontainer after the sleep period?